### PR TITLE
Add io.github.insilmaril.vym

### PR DESCRIPTION
**Pull request by bztd**
### Please confirm your submission meets all the criteria

- [x] Please describe the application briefly.
  Tool for creating and manipulating mind maps.

- [x] The domain used for the application ID is [controlled by the application developers][appid-domain] and the [application id guidelines][appid] are followed.
- [x] I have read the and followed all the [Submission and licence requirements][reqs].
- [x] I have [built][build] and tested the submission locally.
- [x] I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about this submission. https://github.com/insilmaril/vym/issues/67

[appid-domain]: https://docs.flathub.org/docs/for-app-authors/requirements/#control-over-domain-or-repository
[appid]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id
[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission

### CODE REVIEW

**Comment by bztd** [comment]
bot, build io.github.insilmaril.vym

**Comment by bztd** [comment]
bot, build io.github.insilmaril.vym

File: io.github.insilmaril.vym.appdata.xml
Comment (bbhtt) [inline io.github.insilmaril.vym.appdata.xml:1]: Was this submitted upstream?

Comment (bztd) [reply io.github.insilmaril.vym.appdata.xml:1]: I already made the request

File: io.github.insilmaril.vym.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+command: vym
+rename-desktop-file: vym.desktop
+rename-icon: vym #128x128
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+  - --share=ipc
+  - --filesystem=xdg-documents
+  - --own-name=org.insilmaril.*
```
Comment (hfiguiere) [inline io.github.insilmaril.vym.yml:14]: this should be changed upstream to use the same as the app-id. Registration of dbus name have similar constraint.

File: io.github.insilmaril.vym.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+modules:
+  - name: vym
+    buildsystem: cmake-ninja
+    post-install:
+      - install -Dm644 icons/vym-128x128.png /app/share/icons/hicolor/128x128/apps/vym.png
```
Comment (hfiguiere) [inline io.github.insilmaril.vym.yml:20]: this isn't installed by upstream? It's a bug and should be fixed upstream too.

File: io.github.insilmaril.vym.yml [hunk 3]
```diff
@@ -0,0 +1,28 @@
+app-id: io.github.insilmaril.vym
+runtime: org.kde.Platform
+runtime-version: "5.15-23.08"
+sdk: org.kde.Sdk
+command: vym
+rename-desktop-file: vym.desktop
+rename-icon: vym #128x128
```
Comment (hfiguiere) [inline io.github.insilmaril.vym.yml:7]:
```suggestion
rename-icon: vym
```

File: io.github.insilmaril.vym.yml (hunk 2 above)
Comment (bztd) [reply io.github.insilmaril.vym.yml:20]: I mentioned it to the developer

**Comment by bztd** [comment]
bot, build io.github.insilmaril.vym

**Pull request closed without merging**